src/promptflow/promptflow/_sdk/schemas/_run.py

```diff
@@ -48,6 +48,23 @@ def _validate(self, value):
             )
 
 
+class RemoteFlowStr(fields.Str):
```

Comment:
@0mza987 please help check if this works as expected

Reply:
In error message, "azureml:xxx" -> "azureml:<flow-name>"? And there seems to be a typo here:
![image](https://github.com/microsoft/promptflow/assets/2572521/f1fbd61e-798c-4b81-8bdc-d70e903e07ce)

Reply:
Updated